Essentials/src/config.yml

```diff
@@ -573,6 +573,10 @@ npcs-in-balance-ranking: false
 # This is useful when a sign sells or buys one item at a time and the player wants to sell a bunch at once.
 allow-bulk-buy-sell: true
 
+# Allow selling of items with custom names with the /sell command.
+# This may be useful to prevent players accidentally selling named items.
+allow-sell-named-items: false
```

Comment:
`sell-named-items` or `allow-selling-named-items` would sound less awkward in my opinion.